src/events/event_manager.cpp

```diff
@@ -26,6 +26,7 @@ namespace Bald {
     }
 
     void EventManager::Flush(int n) noexcept {
+        CORE_LOG_INFO("[EventManager] About to begin flushing...");
```

Comment:
I would change this to simply "Flushing events"

Reply:
Or if we follow convention -> Add one more log after flush was finished successfully.